script.js

```diff
@@ -0,0 +1,115 @@
+import { filmsSet } from "./data/test-data.js";
+
+const header = {
+  css: "header",
+  paddingX: 20,
+  height: 50,
+  cols: [
+    {
+      view: "label",
+      label: "My App",
+      gravity: 0.2,
+      minWidth: 200
+    },
+    {},
+    {
+      view: "button",
+      type: "icon",
+      icon: "mdi mdi-account",
+      label: "Profile",
+      gravity: 0.2,
+      minWidth: 200
```

Comment:
Not consistent width of button. You can check it on hover
![image](https://user-images.githubusercontent.com/47447469/75444322-49f39500-5974-11ea-840a-befa526f3ba7.png)

Reply:
consistent with what?
made less.